python/lsst/obs/base/formatters/filter.py

```diff
@@ -138,17 +138,4 @@ def _toBytes(self, inMemoryDataset: Any) -> bytes:
         filter["name"] = inMemoryDataset.getName()
         filter["aliases"] = inMemoryDataset.getAliases()
 
-        # This can fail if the filter is no longer available in the
-        # Singleton
-        try:
-            filterProperty = inMemoryDataset.getFilterProperty()
-        except LookupError:
-            pass
-        else:
-            properties = {}
-            properties["lambdaEff"] = filterProperty.getLambdaEff()
-            properties["lambdaMax"] = filterProperty.getLambdaMax()
-            properties["lambdaMin"] = filterProperty.getLambdaMin()
-            filter["properties"] = properties
-
```

Comment:
I don't think it's safe to remove this code yet. If somebody does still need `FilterProperty` (e.g., third-party code), this change to the persistence format will break things.

Reply:
I think this can be removed. I only use canonicalName from the YAML on read.